interop-testing/src/test/java/io/grpc/testing/integration/ProxyTest.java

```diff
@@ -61,7 +61,7 @@ public void shutdownTest() throws IOException {
 
   @Test
   public void smallLatency()
-      throws UnknownHostException, IOException, InterruptedException, ExecutionException {
+      throws IOException, InterruptedException, ExecutionException {
```

Comment:
ok, got your idea, will be reverted. And how about java doc issue, may I fix it or what?